src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ArrayFW.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2016-2019 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.maven.plugin.internal.generated;
+
+import java.util.function.Consumer;
+
+import org.agrona.DirectBuffer;
+import org.reaktivity.reaktor.internal.test.types.Flyweight;
+
+public abstract class ArrayFW<T extends VariantFW> extends Flyweight
+{
+    public abstract int length();
+
+    public abstract int fieldCount();
+
+    public abstract void forEach(Consumer<T> consumer);
+
+    public abstract DirectBuffer items();
+
+    public abstract static class Builder<B extends VariantFW.Builder<O, ?, V>,
+        O extends Flyweight, V extends VariantFW<O, ?>, T extends ArrayFW<V>> extends Flyweight.Builder<T>
+    {
+        private final B itemRW;
```

Comment:
Please Make this `itemRW` field `protected final` and remove the `itemRW()` method.